contracts/src/v0.8/automation/dev/v2_3/AutomationRegistrar2_3.sol

```diff
@@ -73,6 +76,7 @@ contract AutomationRegistrar2_3 is TypeAndVersionInterface, ConfirmedOwner, IERC
 
   struct PendingRequest {
     address admin;
+    IERC20 billingToken;
     uint96 balance;
   }
```

Comment:
I think we could get rid of this entire struct and change `s_pendingRequests` to...
```solidity
mapping(bytes32 => PendingRequest) private s_pendingRequests;
```
... this is what our friends recommended too :)

Reply:
hmm, can you elaborate a bit more on this? if we dont have this PendingRequest struct, when users cancel their pending upkeep, how we do refund?

Reply:
actually nevermind - forget I said anything 🙃